src/net/toxme.cpp

```diff
@@ -319,13 +363,9 @@ QString Toxme::getErrorMessage(int errorCode)
         return QObject::tr("Name not found");
     case -31:
         return QObject::tr("Tox ID not sent");
-    case -41:
-        return QObject::tr("Lookup failed because the server replied with invalid data");
     case -42:
         return QObject::tr("That user does not exist");
-    case -43:
-        return QObject::tr("Internal lookup error. Please file a bug");
     default:
-        return QObject::tr("Unknown error (%1)").arg(errorCode);
+        return QString();
```

Comment:
I'd prefer if the default message returned something like "Internal ToxMe error" and not an empty string.